README.md

```diff
@@ -140,10 +138,9 @@ You can deploy this project using: GitHub Pages
 
```

Comment:
![image](https://github.com/Grandi0z/Math_magicians/assets/93039359/b3feabae-2bc7-4d5b-90da-2b8a28531f97)

- [ ] It would be nice if you fix your linkedin link because it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍